modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/resource_selector/page.jsp

```diff
@@ -25,7 +25,7 @@ String warningMessage = (String)request.getAttribute("liferay-frontend:resource-
 	<span aria-hidden="true" class="loading-animation"></span>
 
 	<react:component
-		module="{ResourceSelector} from frontend-taglib"
+		module="resource_selector/ResourceSelector"
 		props='<%=
```

Comment:
I think this was needed to use ESM. @bryceosterhaus could you check this?

Reply:
This was due to a problem rebasing with IJ because i did the change previously to use ESM 😞 So you're right, it should be `module="{ResourceSelector} from frontend-taglib"`